tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -114,6 +114,7 @@ Verify That Blackbox-exporter Image Is A CPaaS Built Image
     [Tags]    Sanity
     ...       Tier1
     ...       ODS-735
+    Skip If RHODS Is Self-Managed
```

Comment:
instead of skipping, should we check that these deployments are not present in the cluster?

Reply:
Yes, that will be in another PR. I wanted to do something quick for today

Reply:
okk